docs/tools/idx/quickstart.md

```diff
@@ -0,0 +1,99 @@
+# Quickstart
+
+Get started exploring what's possible with IDX using the [IDX SDK](../reference/idx.md). For usage in a Command Interface, you will need to [install the IDX CLI](../reference/cli.md).
+
+!!! warning ""
+
+    :octicons-alert-16: IDX is in alpha, Libraries may be unstable and APIs are subject to change. Data created on IDX during alpha will *not* be portable to production. Please share what you're working on and report any issues in the [IDX Discord](https://chat.idx.com){:target="_blank"}.
+
+!!! warning ""
+    :octicons-stop-16: If you haven't completed the Ceramic quickstart some information here may not make complete sense. We recommend coming here after you've completed the [Ceramic JavaScript Client Quickstart](../../build/javascript/quick-start.md).
+
+## **Prerequisites**
+
+The IDX SDK requires [Node.js](https://nodejs.org/){:target="_blank"} v14+ and npm v6+ (usually installed with Node.js). Make sure both are installed.
```

Comment:
```suggestion
The IDX SDK requires [Node.js](https://nodejs.org/){:target="_blank"} v14 and npm v6 (usually installed with Node.js). Make sure both are installed.
```

Reply:
I'm pretty sure today Ceramic only works on *exactly* node v14 and npm v6.  Pretty sure we've had issues reported by the community using newer versions.

Reply:
I think that it works on other versions, but it needs some extra config. I'm not opposed to making the change regardless and having it a bit more specific will minimize the questions users have.